content/en/docs/spinnaker-user-guides/pacrd.md

```diff
@@ -486,6 +486,7 @@ stages:
 # ...
 ```
 
+- Pipeline names do not support spaces and must follow the [Kubernetes naming convention](https://v1-16.docs.kubernetes.io/docs/concepts/overview/working-with-objects/names/#names).  "Kubernetes resources can have names up to 253 characters long. The characters allowed in names are: digits (0-9), lower case letters (a-z), -, and .."
```

Comment:
@away168 I see you are linking to Kubernetes v1.16 documentation, which will be deprecated in 2 releases. The Names section you link to changed in v1.18. So, do naming conventions for pipelines depend on the version of Kubernetes being used?  or are the naming conventions the same regardless of Kubernetes version?   

If PaCRD enforces specific naming conventions regardless of K8s version, we should state the convention without referring to K8s.     Otherwise we should state that pipeline names following the naming conventions for the k8s version you are using and point to  https://kubernetes.io/docs/concepts/overview/working-with-objects/. 

Let me know. I'm happy to update the text.

Reply:
@dogonthehorizon any thoughts on this?

Reply:
@dogonthehorizon 

> LGTM pending linking to the correct K8s doc link
I have some specific questions:  _So, do naming conventions for pipelines depend on the version of Kubernetes being used? or are the naming conventions the same regardless of Kubernetes version?_